ros2pkg/ros2pkg/resource/ament_cmake/CMakeLists.txt.em

```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.5)
+project(@(project_name))
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+@[if cpp_library_name]@
+find_package(ament_cmake_ros REQUIRED)
+@[end if]@
+@[if dependencies]@
+@[  for deb in dependencies]@
+find_package(@deb REQUIRED)
+@[  end for]@
+@[else]@
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+@[end if]@
+@[if cpp_library_name]@
+
+include_directories(
+  include
+)
+
+add_library(${PROJECT_NAME} src/@(cpp_library_name))
```

Comment:
Similar to the comment about the executable name. I would expect to get a target with the name passed in with `--cpp-library-name`.